src/blockChainManager/block.ts

```diff
@@ -0,0 +1,5 @@
+export interface block {
+    id: string;
+    hashId: string;
```

Comment:
rename properties like this :  
id => position
hashId => id
previousnodeid => previousBlockId